vms/platformvm/block/executor/acceptor.go

```diff
@@ -33,11 +33,11 @@ type acceptor struct {
 }
 
 func (a *acceptor) BanffAbortBlock(b *block.BanffAbortBlock) error {
-	return a.abortBlock(b, "banff abort")
+	return a.optionBlock(b, "banff abort")
 }
 
 func (a *acceptor) BanffCommitBlock(b *block.BanffCommitBlock) error {
-	return a.commitBlock(b, "apricot commit")
+	return a.optionBlock(b, "banff commit")
```

Comment:
This was the wrong block previously

Reply:
Are you referring to using the name `apricot` or the call to `commitBlock` instead of `optionBlock` ?